recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
Hooks won't allow shebang

Reply:
They do, see the previous success message 2 hours ago

Reply:
@uilianries why this was not detected by the hook?